source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/en.lproj/ACRUIButton.xib

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I may be missing something, but where is this xib used?

Reply:
thanks! I think it got added accidentally.

---
In reply to: [195572148](https://github.com/Microsoft/AdaptiveCards/pull/1474#discussion_r195572148) [](ancestors = 195572148)